crates/cheatable-starknet/src/cheatcodes/mock_call.rs

```diff
@@ -0,0 +1,35 @@
+use crate::{cheatcodes::EnhancedHintError, CheatedState};
+use starknet_api::core::{ContractAddress, EntryPointSelector};
+use starknet_api::hash::StarkFelt;
+use std::collections::HashMap;
+
+impl CheatedState {
+    pub fn start_mock_call(
+        &mut self,
+        contract_address: ContractAddress,
+        function_name: EntryPointSelector,
+        ret_data: Vec<StarkFelt>,
+    ) -> Result<(), EnhancedHintError> {
+        self.mocked_functions.entry(contract_address).or_insert_with(|| HashMap::new());
```

Comment:
Doesn't it return `contract_mocked_functions` already? This way we could get rid of the unwrap below